src/js/pages/system/OverviewDetailTab.js

```diff
@@ -66,14 +71,39 @@ class OverviewDetailTab extends mixin(StoreMixin) {
       {
         name: "metadata",
         events: ["dcosBuildInfoChange", "dcosSuccess", "success"]
-      },
-      {
-        name: "state",
-        events: ["success"],
-        listenAlways: false
       }
     ];
 
+    request({ type: "GET_FLAGS" }).subscribe(message => {
```

Comment:
@nLight wondering if we should have some error handling/retry here? Looks like we don't have any for the `mesos-client` `request` and `subscribe` methods.

Reply:
Do you think we should add like `retry(3)` just to be on a safe side?

Reply:
I think we should and we should really show an error message after that. We need to architect our request error handling a little e.g. how we deal with 401, 403 vs 500's. A retry on 403 is not smart. cc/ @orlandohohmeier

Reply:
BTW - I don't think it should block this PR.

Reply:
👍  https://jira.mesosphere.com/browse/DCOS-19443